astro/src/components/docs/release-notes/ReleaseNotesIssue.astro

```diff
@@ -1,15 +1,22 @@
 ---
+// Creates an issue entry in a release notes section, including the "Resolves GitHub issue #nnnn" bit
+//
+// issue: a comma-separated list of GitHub issue numbers
+
 export interface Props {
   issue: string;
 }
 
 const { issue } = Astro.props as Props;
+const issueNumbers = issue.split(',');
 ---
 <Fragment>
 <slot></slot>
 <ul>
-  <li>
-    Resolves <a target="_" href={`https://github.com/FusionAuth/fusionauth-issues/issues/${issue}`}>GitHub Issue #{issue}</a>
-  </li>
+  {issueNumbers.map((inum) => (
+    <li>
+      Resolves <a target="_" href={`https://github.com/FusionAuth/fusionauth-issues/issues/{inum}`}>GitHub Issue {inum}</a>
```

Comment:
Is the `$` not necessary for the interpolated string? I think we want to keep the `#` before the issue number in the text.

```suggestion
      Resolves <a target="_" href={`https://github.com/FusionAuth/fusionauth-issues/issues/${inum}`}>GitHub Issue #{inum}</a>
```